ProcessMaker/processmaker
# Create 'Publish Screen Templates' User Permissions

This PR adds the 'publish-screen-templates' permission for users.

## Solution
- Create migration to add new user permission
- Update PermissionsSeeder to include new permission

## How to Test
1. Navigate to Admin -> Users
2. Select or Create a user
3. View the Permissions tab
4. Navigate to Users
5. Ensure the Publish Screen Templates permission is visible. It should be deselected by default.

## Related Tickets & Packages
- [FOUR-13549](https://processmaker.atlassian.net/browse/FOUR-13549)

ci:next

.
## Code Review Checklist


[FOUR-13549]: https://processmaker.atlassian.net/browse/FOUR-13549?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
